vowpalwabbit/automl.cc

```diff
@@ -78,6 +78,18 @@ void fail_if_enabled(vw& all, const std::set<std::string>& not_compat)
     if (not_compat.count(reduction) > 0) THROW("Error: automl does not yet support this reduction: " + reduction);
   }
 }
+
+std::string interaction_vec_t_to_string(const std::vector<std::vector<namespace_index>>& interactions)
+{
+  std::stringstream ss;
+  for (const std::vector<namespace_index>& v : interactions)
+  {
+    ss << "-q ";
```

Comment:
skip constant namespace / wildcard

Reply:
nit: maybe make it extendable for when higher order interactions are added and instead of `-q` use some other separator?